src/qml/qgismobileapp.qml

```diff
@@ -932,6 +939,7 @@ ApplicationWindow {
       onLoadProjectEnded: {
         busyMessage.visible = false
         openProjectDialog.folder = qgisProject.homePath
+        mapCanvasBackground.color = mapCanvas.mapSettings.backgroundColor
```

Comment:
Does it help if this line is removed?

```suggestion
        mapCanvasBackground.color = mapCanvas.mapSettings.backgroundColor
```

What this does is overwriting line 167 with a fixed value and "destroying" the dynamic binding.

Reply:
Sorry, I meant that with line 167 alone, the background color doesn't dynamically switch. Let me try again to make sure I'm not just plain wrong.

Reply:
Yep, just re-confirmed. The dynamic binding doesn't work, that line in onLoadProjectedEnded is needed.

Reply:
One more to add for the "let's get rid of all the warnings" project.
Ther's this hidden in the log

> qrc:/qml/qgismobileapp.qml:167: ReferenceError: mapSettings is not defined

But even now I can't get it to work dynamically, weird. Let's stick to the current solution if no one finds something.